### PR TITLE
Hot fix rrp slow

### PR DESCRIPTION
**Pull request by padhupradheep**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | #2332  |
| Primary OS tested on | Ubuntu |
| Robotic platform tested on | mp-400 from Neobotix |

---

## Description of contribution in a few bullet points

* Fixed the segfault issue that occurred when the robot was set to drive at high speed using RPP. 

## Description of documentation updates required from your changes

NA

---

## Future work that may be required in bullet points

NA

#### For Maintainers: 
- [ ] Check that any new parameters added are updated in navigation.ros.org
- [ ] Check that any significant change is added to the migration guide
- [ ] Check that any new functions have Doxygen added
- [ ] Check that any new features have test coverage
- [ ] Check that any new plugins is added to the plugins page
- [ ] If BT Node, Additionally: add to BT's XML index of nodes for groot, BT package's readme table, and BT library lists

### CODE REVIEW

File: nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp [hunk 1]
```diff
@@ -450,7 +459,16 @@ bool RegulatedPurePursuitController::inCollision(const double & x, const double
 double RegulatedPurePursuitController::costAtPose(const double & x, const double & y)
 {
   unsigned int mx, my;
-  costmap_->worldToMap(x, y, mx, my);
+
+  if (!costmap_->worldToMap(x, y, mx, my)) {
+    RCLCPP_FATAL(
+      logger_,
+      "The dimensions of the costmap is too small to fully include your robot's footprint"
```
Comment (SteveMacenski) [inline nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp:466]: Needs a space and a comma, the line extension doesn't add a space between the words

Comment (SteveMacenski) [reply nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp:466]: as well below

Comment (SteveMacenski) [reply nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp:466]: and above

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

**Comment by SteveMacenski** [comment]
@padhupradheep last thing here is just string formatting, then good to go!

**Comment by padhupradheep** [comment]
> @padhupradheep last thing here is just string formatting, then good to go!

Done @SteveMacenski

**Review by SteveMacenski — APPROVED** [review]

**Comment by SteveMacenski** [comment]
Thanks!!

**Pull request merged**